### ISSUE TITLE
Missing aria-controls in Bootstrap v3 examples for Dropdowns

### ISSUE THREAD

**Issue by PowerGamer1** [twbs/bootstrap#20375]
In the docs for Bootstrap 3 examples  `aria-` attributes are used extensively. I think all Dropdown examples (maybe other examples too) are missing `aria-controls` attribute. For example (http://getbootstrap.com/components/#dropdowns-example):
```HTML
<div class="dropdown">
  <button class="btn btn-default dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
    Dropdown
    <span class="caret"></span>
  </button>
  <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
    <li><a href="#">Action</a></li>
    <li><a href="#">Another action</a></li>
    <li><a href="#">Something else here</a></li>
    <li role="separator" class="divider"></li>
    <li><a href="#">Separated link</a></li>
  </ul>
</div>
```
The `button` element in the example having `aria-expanded` attribute should also have `aria-controls` attribute set: a quote from http://www.w3.org/TR/wai-aria/states_and_properties#aria-expanded:

> If the element with the `aria-expanded` attribute controls the expansion of another grouping container that is not 'owned by' the element, the author SHOULD reference the container by using the `aria-controls` attribute.

**Comment by cvrebert** [comment]
CC: @patrickhlauke

**Comment by caseyjhol** [comment]
It looks like proper usage is for the list element to have the `aria-expanded` attribute, and for the button to have `aria-controls`. i.e. the button _should not_ have the `aria-expanded` attribute.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
